cluster-autoscaler/cloudprovider/hetzner/hcloud-go/hcloud/action.go

```diff
@@ -0,0 +1,226 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
It seems like that `hcloud-go` is a copy from https://github.com/hetznercloud/hcloud-go . It's licensed under MIT with "License and copyright notice". I'm not sure that hcloud-go can be directly copy&pasted with a license change.